packages/web-runtime/src/index.ts

```diff
@@ -60,8 +63,13 @@ export const bootstrapApp = async (configurationPath: string): Promise<void> =>
   })
   const themePromise = announceTheme({ store, app, designSystem, runtimeConfiguration })
   await Promise.all([applicationsPromise, themePromise])
+  const customTranslations = await loadCustomTranslations({ configurationManager })
```

Comment:
Last nitpick: tiny performance improvement possible by splitting this into promise creation and awaiting the resolved promise together with other promises. This is a good thing because the promise execution starts immediately with creation, so this enables parallel processing instead of awaiting promises sequentially.

In other words:
Please split this line into:
```js
const customTranslationsPromise = loadCustomTranslations({ configurationManager })
```
for promise creation and add that to the `Promise.all`. You can access the resolved custom translations like this:

```js
const [customTranslations] = await Promise.all([customTranslationsPromise, applicationsPromise, themePromise])
```